apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -138,23 +149,42 @@ export function InstructorQuestionSettings({
                     slashes to separate directories.
                   </small>
                 </div>
-
-                <div class="table-responsive card mb-3">
+                <div class="table-responsive card mb-3 overflow-visible">
                   <table
                     class="table two-column-description"
                     aria-label="Question topic, tags, and assessments"
                   >
                     <tr>
-                      <th class="border-top-0">Topic</th>
-                      <td class="border-top-0">${TopicBadge(resLocals.topic)}</td>
+                      <th class="align-middle">Topic</th>
+                      <!-- The style attribute is necessary until we upgrade to Bootstrap 5.3 -->
+                      <!-- This is used by tom-select to style the active item in the dropdown -->
+                      <td style="--bs-tertiary-bg: #f8f9fa">
```

Comment:
I just pushed a commit adding this. This is needed because `tom-select` relies on a variable from Bootstrap 5.3 (`--bs-tertiary-bg`).